src/pages/publication/references_tab/JournalTypeForm.tsx

```diff
@@ -0,0 +1,33 @@
+import { FormikProps, useFormikContext } from 'formik';
+import React, { FC } from 'react';
+import { JournalPublication } from '../../../types/publication.types';
+import { ReferenceFieldNames, JournalType } from '../../../types/publicationFieldNames';
+import SelectTypeField from './components/SelectTypeField';
+import JournalArticleForm from './sub_type_forms/JournalArticleForm';
+import JournalForm from './sub_type_forms/JournalForm';
+import styled from 'styled-components';
+
+const StyledSelectContainer = styled.div`
```

Comment:
flytte ut denne, så vi slipper å lage den på nytt for hver type?
vi har en pent brukt `styled` mappe vi tenkte å legge ting, men vi kan like godt bare flytte den til `reference_tab` folder for min del siden den bare skal brukes her uansett

Reply:
jau, det kan vi godt gjøre 👍